src/Sarif/Core/SarifLog.cs

```diff
@@ -101,8 +101,20 @@ public static SarifLog Load(Stream source, bool deferred = false)
                 return false;
             }
 
-            await Post(postUri, new MemoryStream(fileBytes), httpClient);
-            Console.WriteLine($"Posted log file successfully to: {postUri}");
+            HttpResponseMessage response = await Post(postUri, new MemoryStream(fileBytes), httpClient);
+            string responseText = response.Content.ReadAsStringAsync().Result;
+
+            try
+            {
+                response.EnsureSuccessStatusCode();
+            }
+            catch (Exception)
+            {
+                Console.WriteLine($"Error Posting log file to: {postUri}. Endpoint provided message: {responseText}");
+                return false;
+            }
+
```

Comment:
It seems better to check response.IsSuccessStatusCode.

Reply:
That is, as opposed to doing a try/catch here.

Reply:
And I also think status code is useful to user, I think we should add it to the output as well